core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -780,6 +780,14 @@ public String toString() {
               + "is set to 0, the cache will be disabled, and "
               + "`alluxio.user.file.metadata.load.type=Once` will behave like `Always`.")
           .build();
+  public static final PropertyKey MASTER_UFS_BLOCK_LOCATION_CACHE_CAPACITY =
+      new Builder(Name.MASTER_UFS_BLOCK_LOCATION_CACHE_CAPACITY)
+          .setDefaultValue(1000000)
+          .setDescription("The capacity of the UFS block locations cache. "
+              + "This cache caches UFS block locations for files that are persisted "
+              + "but not in Alluxio space, so that listing status of these files do not need to "
+              + "repeatedly ask UFS for their block locations.")
```

Comment:
Should we have size 0 mean disabled cache?

Reply:
make sense, I'll add it.

Reply:
From http://google.github.io/guava/releases/snapshot/api/docs/com/google/common/cache/CacheBuilder.html#maximumSize-long-
```If maximumSize is zero, elements will be evicted immediately after being loaded into cache. This can be useful in testing, or to disable caching temporarily.```

So if the size is 0, the cache is disabled by default.

Reply:
I added description